src/IO.cpp

```diff
@@ -14,8 +18,33 @@ namespace ExaEpi
 namespace IO
 {
 
-void writePlotFile (const AgentContainer& pc, const iMultiFab& /*num_residents*/, const iMultiFab& unit_mf,
-                    const iMultiFab& FIPS_mf, const iMultiFab& comm_mf, const int step) {
+/*! \brief Write plotfile of computational domain with disease spread and census data at a given step.
+
+    Writes the current disease spread information and census data (unit, FIPS code, census tract ID,
+    and community number) to a plotfile:
+    + Create an output MultiFab (with the same domain and distribution map as the particle container)
+      with 9 components:
+      + component 0: total
+      + component 1: never infected (#Status::never)
+      + component 2: infected (#Status::infected)
+      + component 3: immune (#Status::immune)
+      + component 4: previously infected (#Status::susceptible QDG??)
```

Comment:
@atmyers @tannguyen153 Is the Status::susceptible same as "previously infected" in the plotfile?